cephci/utils/configure.py

```diff
@@ -314,7 +314,7 @@ def add_images_to_private_registry(
 
     for image in images:
         img_reg = image.replace(f"{registry}/", "")
-        src_image = f"{image}"
+        src_image = f"{registry}/{image}"
```

Comment:
same as above

Reply:
So looks like this was missed earlier, the ideal way as per 
https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/5/html-single/installation_guide/index#configuring-a-private-registry-for-a-disconnected-installation_install is as above Vaibhav